app/players/elves/druid.py

```diff
@@ -0,0 +1,18 @@
+from app.players.elves.elf import Elf
+
+
+class Druid(Elf):
+    def __init__(
+            self, nickname: str,
+            musical_instrument: str,
+            favourite_spell: str
+    ) -> None:
+        super(Druid, self).__init__(nickname, musical_instrument)
```

Comment:
why did you decide to send `Druid, self` into `super` here? why did you decide to do it only here? please, keep one codestyle everywhere

Reply:
It was said that best practice not to add it, but Pycharm adds it, so in other cases i deleted it. My bad, didn't see that, corrected